neurokit2/hrv/intervals_process.py

```diff
@@ -5,15 +5,19 @@
 
 from ..misc import NeuroKitWarning
 from ..signal import signal_detrend, signal_interpolate
-from .hrv_utils import _intervals_sanitize
+from .intervals_utils import (
+    _intervals_sanitize,
+    _intervals_time_to_sampling_rate,
+    _intervals_time_uniform,
+)
 
 
 def intervals_process(
     intervals,
     intervals_time=None,
     interpolate=False,
     interpolation_rate=100,
-    detrend_method=None,
+    detrend=None,
```

Comment:
should we detrend by default tho?

Reply:
No I wouldn't, since it would be a major change from the current implementation and also it shouldn't be done for all hrv features (e.g. MeanNN, MaxNN)